Changelog.md

```diff
@@ -9,6 +9,8 @@ Compiler Features:
 
 Bugfixes:
  * SMTChecker: Fix SMT logic error when assigning to an array of addresses.
+ * Yul AST: Fix shifted native source locations when debug info selection included code snippets.
```

Comment:
Not sure if we want to have a changelog entry for this one. Technically, we never made Yul AST output non-experimental. On the other hand, the feature has been there for a while and it's still nice to have a record of what changed.

Reply:
I think it is good to have some documentation, even if the feature is experimental. Could be worth clarifying in the changelog that it is indeed an experimental feature, though?

Reply:
Yeah, I think it is ok to have a changelog entry here, since we also had it when added the feature in https://github.com/ethereum/solidity/pull/14177/files#diff-c275fec9c453c2a42515bc5ab47e30fa4130ff1426aef2f6e000a9a34e122cb8R8

Reply:
> since we also included it when the feature was added in 

Yeah, but that was actually an oversight back then :) But given that it was not marked as experimental, perhaps this one is fine not being marked as such either...